README.md

```diff
@@ -45,8 +45,10 @@ first, and then:
 
 ```bash
     # Create and activate myenv conda environment (optional, but recommended)
-    conda create -n myenv
-    source activate myenv
+    conda create -n myenv python=3.6
+    conda activate myenv
+    # Install dependencies using conda
+    conda install --yes --file requirements.txt
```

Comment:
I am not sure this line is needed. This is installing from conda (not source), so `conda install -c theochem procrustes` should take care of installing/updating dependencies.

Reply:
Nice catch! This is unnecessary and has been fixed in 0045283.